src/ramptr.c

```diff
@@ -0,0 +1,23 @@
+#include "global.h"
+#include "ramptr.h"
+#include "data.h"
+#include "functions.h"
+
+void sub_08012C44(void)
+{
+    CpuFill16(0, gUnk_03006590, 0x218);
+    gUnk_030067B0 = (u32*)EWRAM_START;
```

Comment:
I think for now we should add `gUnk_02000000` to linker.ld unless not matching.

Reply:
It is not matching if I use gUnk_02000000 instead of this cast. It loads the symbol from the pool.